app/src/store/store.ts

```diff
@@ -1,80 +1,34 @@
 import Vue from 'vue';
-import Vuex from 'vuex';
+import Vuex, {StoreOptions} from 'vuex';
 import DataService from '../utils/services/DataService';
 import LocationService from '@/utils/services/LocationService';
 import router from '@/router';
-import Tag from '@/models/tag';
 import Post from '@/models/post';
+import textSearchModule from './TextSearch';
+import locationSearchModule from './LocationSearch';
+import {RootState} from './types';
 Vue.use(Vuex);
 
-const store = new Vuex.Store({
+const store: StoreOptions<RootState> = {
+  modules: {
+    textSearchModule,
+    locationSearchModule,
+  },
   state: {
-    searchProposals: [
-      { header: 'Vorschläge' },
-      { divider: true },
-    ],
-    radii: [
-      {
-        text: 'Überall',
-        value: '',
-      },
-      {
-        text: '5 km',
-        value: '5km',
-      },
-      {
-        text: '10 km',
-        value: '10km',
-      },
-      {
-        text: '25 km',
-        value: '25km',
-      },
-      {
-        text: '50 km',
-        value: '50km',
-      },
-    ],
     posts: [] as Post[],
-    labels: [] as string[],
-    synonyms: [] as string[],
-    searchValues: [] as string[],
-    locationSearchValue: '',
-    radiusSearchValue: '',
-    selectedLocation: '',
-    selectedTag: '',
     page: 1 as number
-  },
+  } as RootState,
   mutations: {
-    addSearchValue(state, value: string): void {
-      state.searchValues.push(value);
-    },
-    initializeSearchProposals(state, tags: []): void {
-      state.searchProposals = state.searchProposals.concat(tags);
-    },
-    removeSearchValue(state, value): void {
-      state.searchValues.splice(state.searchValues.indexOf(value), 1);
+
```

Comment:
it would probably be overkill to exclude "root" mutations in a separate file like it was done with TextSearch etc  ?

Reply:
Yeah I thought about that. I think it could be a bit confusing...